projects/packages/publicize/src/class-rest-controller.php

```diff
@@ -454,6 +454,9 @@ public function share_post( $request ) {
 		$post_id             = $request->get_param( 'postId' );
 		$message             = trim( $request->get_param( 'message' ) );
 		$skip_connection_ids = $request->get_param( 'skipped_connections' );
+		$async               = $request->get_param( 'async' );
+
+		$is_async_share = isset( $async ) && true === $async;
```

Comment:
I think we can simply use this

```suggestion
		$async               = (bool) $request->get_param( 'async' );
```

Then update the variable name below.